### ISSUE TITLE
A better alternative to convnetjs

### ISSUE THREAD

**Issue by TrevorBlythe** [karpathy/convnetjs#122]
I made a library similar to this. 
This library has Deconv layers and other new features that not even convnetjs has. It also let's you utilize your GPU.
It also doesn't require you to put your data in a custom data type.

<a href="https://github.com/TrevorBlythe/MentisJS">Link</a>

https://github.com/TrevorBlythe/MentisJS/

**Comment by xieguigang** [comment]
WOW, and by the way, are there plans for implementing BatchNorm? [BatchNorm layer](https://github.com/karpathy/convnetjs/issues/117)

**Comment by 371148606** [comment]
您好，我是薛文毅，已收到您的邮件，一会查收，谢谢！！

**Comment by TrevorBlythe** [comment]
> WOW, and by the way, are there plans for implementing BatchNorm? [BatchNorm layer](https://github.com/karpathy/convnetjs/issues/117)


Yeah I can make batch norm! I'll start it this next week since I'm off doing stuff rn but that sounds like a good idea.
